ansible/roles/edu-sharing/vars/versions/7.0.0.yml

```diff
@@ -0,0 +1,20 @@
+---
+edu_sharing_archive_url: https://artifacts.edu-sharing.com/repository/maven-remote/org/edu_sharing/edu_sharing-projects-community-deploy-docker-compose/7.0.0/edu_sharing-projects-community-deploy-docker-compose-7.0.0-bin.zip
+
+# Enable/Disable migration from previous Edu-Sharing version
+# This must run once and then please make it false
+edu_sharing_enable_migration: true
```

Comment:
Most probably users won't be aware of this and then the migration will run at every update. We should automatically detect if the migration was already processed and only run the task, if the migration is necessary (existing old version and not processed yet in new version)

Reply:
Hello @mirjan-hoffmann 

I created another solution, can you have a look